### PR TITLE
feat: added seed file for bid_status_histories

### LINKED ISSUE #53: Seed script for bid_status_histories table🌱
## Objective
Create a comprehensive seed script for the bid_status_histories with realistic data.

## Requirements
- Link to existing bid requests

## Implementation Checklist
- [ ] Write SQL seed script
- [ ] Include proper status
- [ ] Add meaningful notes for each status change
- [ ] Test data consistency and relationships

## Considerations
- Use uuid_generate_v4() for ID generation

## SQL Table
```sql
CREATE TABLE bid_status_histories (
    id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
    bid_request_id UUID REFERENCES bid_requests(id) ON DELETE CASCADE,
    status TEXT NOT NULL,
    notes TEXT,
    changed_by TEXT REFERENCES users(id) ON DELETE SET NULL,
    created_at TIMESTAMP WITH TIME ZONE DEFAULT NOW()
);
```

## Acceptance Criteria
A complete SQL seed script for bid_status_histories table

### PR DESCRIPTION
**Pull request by ikezuby2012**
# Pull Request for SafeTrust - Close Issue

❗ **Pull Request Information**

feat: added seed file for bid_status_histories table at seeds/safetrust folder
Add here some information

## 🌀 Summary of Changes

- **Add here the changes**:
- Changes just made to seeds/safetrust (new features)
- **More changes**:

## 🛠 Testing
NOT NEEDED!
### Evidence Before Solution

- **Video**: [Link to Loom video](https://loom.com)

### Evidence After Solution

- **Video**: [Link to Loom video](https://loom.com)

## 📂 Related Issue

This pull request will **close # [Issue Number]** upon merging.

This Pull request will close #53 upon merging
---

🎉 Thank you for reviewing this PR! 🎉

### CODE REVIEW

File: seeds/safetrust/1734448906685_bid_status_histories.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+INSERT INTO bid_status_histories (id, bid_request_id, status, notes, changed_by, created_at)
+VALUES
+    (uuid_generate_v4(), (SELECT id FROM bid_requests LIMIT 1 OFFSET 0),, 'Submitted', 'Bid request has been submitted.', (SELECT id FROM users WHERE username = 'user'), NOW()),
+    (uuid_generate_v4(), (SELECT id FROM bid_requests LIMIT 1 OFFSET 1), 'Reviewed', 'Initial review completed by team lead.', (SELECT id FROM users WHERE username = 'team-lead'), NOW() - INTERVAL '1 day'),
+    (uuid_generate_v4(), (SELECT id FROM bid_requests LIMIT 1 OFFSET 2), 'Approved', 'Bid request approved by manager.', (SELECT id FROM users WHERE username = 'manager'), NOW() - INTERVAL '2 days'),
+    (uuid_generate_v4(), (SELECT id FROM bid_requests LIMIT 1 OFFSET 3), 'In_Progress', 'Work on the bid has commenced.', (SELECT id FROM users WHERE username = 'user'), NOW() - INTERVAL '3 days'),
```
Comment (rvalenciano) [inline seeds/safetrust/1734448906685_bid_status_histories.sql:8]: I would rephrase this as: 

 'Work on the bid has begun.'

File: storage-service/package.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
     "morgan": "^1.10.0",
     "multer": "^1.4.5-lts.1",
     "multer-s3": "^3.0.1",
+    "storage-service": "file:",
```
Comment (rvalenciano) [inline storage-service/package.json:20]: Why are we touching this? Please undo this change.

File: package.json [hunk 3]
```diff
@@ -18,12 +18,13 @@
...
+    "backend": "file:",
+    "cucumber-html-reporter": "^5.5.0",
     "dotenv": "^16.4.5",
     "express": "^4.21.1",
     "express-rate-limit": "^7.4.1",
     "graphql-request": "^7.1.2",
     "nodemailer": "^6.9.16",
-    "winston": "^3.17.0",
-    "cucumber-html-reporter": "^5.5.0"
+    "winston": "^3.17.0"
```
Comment (rvalenciano) [inline package.json:28]: Please undo these changes.

File: seeds/safetrust/1734448906685_bid_status_histories.sql (hunk 1 above)
Comment (ikezuby2012) [reply seeds/safetrust/1734448906685_bid_status_histories.sql:8]: noted

File: package.json (hunk 3 above)
Comment (ikezuby2012) [reply package.json]: Done, please review changes

**Review by sotoJ24 — APPROVED** [review]

**Review by sotoJ24 — APPROVED** [review]

**Review by sotoJ24 — COMMENTED** [review]
Dear @ikezuby2012, the package-lock.json is still modified, can you please undo that's changes, btw we need almost 20 rows to test the pagination, thanks and regards

Screenshot 2024-12-20 195034

**Review by sotoJ24 — CHANGES_REQUESTED** [review]
please, keep us updated about the request changes

**Comment by ikezuby2012** [comment]
done, please review

**Review by sotoJ24 — APPROVED** [review]

**Review by sotoJ24 — CHANGES_REQUESTED** [review]
Hi Dear @ikezuby2012 hope you're doing well

we noticed that when the query is executed, throwing an error:  ` "message": "column \"username\" does not exist"`

Screenshot 2024-12-30 191309

The username column must be replaced with the **email** column for it to work.

Whatever you need just let Us know, thanks

**Comment by ikezuby2012** [comment]
Please review

**Comment by sotoJ24** [comment]
Hi @ikezuby2012, it still doesn't work, because the email table does not exist
Screenshot 2024-12-31 160553

could you make the query something like this: 

> `INSERT INTO bid_status_histories (id, bid_request_id, status, notes, changed_by, created_at) VALUES (uuid_generate_v4(), (SELECT id FROM bid_requests LIMIT 1 OFFSET 0), 'Submitted', 'Bid request has been submitted.', (SELECT id FROM users LIMIT 1 OFFSET 0), NOW()),`  

 and the following,
 Regards

**Review by sotoJ24 — CHANGES_REQUESTED** [review]
Hi @ikezuby2012, it still doesn't work, because the email table does not exist
Screenshot 2024-12-31 160553

could you make the query something like this: 

> `INSERT INTO bid_status_histories (id, bid_request_id, status, notes, changed_by, created_at) VALUES (uuid_generate_v4(), (SELECT id FROM bid_requests LIMIT 1 OFFSET 0), 'Submitted', 'Bid request has been submitted.', (SELECT id FROM users LIMIT 1 OFFSET 0), NOW()),`  

 and the following,
 Regards

**Comment by ikezuby2012** [comment]
check now

**Review by sotoJ24 — APPROVED** [review]

**Review by sotoJ24 — CHANGES_REQUESTED** [review]
Screenshot 2025-01-07 182341
Screenshot 2025-01-07 182312

Hi,

Dear @ikezuby2012  could you please help us to remove those package dependencies from the package-lock.json, please

**Comment by ikezuby2012** [comment]
check again

File: package-lock.json [hunk 4]
```diff
@@ -191,6 +216,22 @@
       "resolved": "https://registry.npmjs.org/cookie-signature/-/cookie-signature-1.0.6.tgz",
       "integrity": "sha512-QADzlaHc8icV8I7vbaJXJwod9HWYp8uCqf1xa4OfNu1T7JVxQIrUgOWtHdNDtPiywmFbiS12VjotIXLrKM3orQ=="
     },
+    "node_modules/cucumber-html-reporter": {
```
Comment (sotoJ24) [inline package-lock.json:219]: Remove  ->   
 ` 
 "node_modules/cucumber-html-reporter": {
      "version": "5.5.1",
      "resolved": "https://registry.npmjs.org/cucumber-html-reporter/-/cucumber-html-reporter-5.5.1.tgz",
      "integrity": "sha512-bXy8aUSUhgFavsB9T4nQLx4B3Q+HBY2Chwqs6bOMOgLKqWSx2VBYjEgzv57TvQvEbrluAiEqP93rDBLimTodnQ==",
      "dependencies": {
        "chalk": "^2.4.2",
        "find": "^0.3.0",
        "fs-extra": "^8.1.0",
        "js-base64": "^2.3.2",
        "jsonfile": "^5.0.0",
        "lodash": "^4.17.11",
        "node-emoji": "^1.10.0",
        "open": "^6.4.0",
        "uuid": "^3.3.3"
      }
    },
`

File: package-lock.json [hunk 5]
```diff
@@ -285,6 +326,14 @@
       "resolved": "https://registry.npmjs.org/escape-html/-/escape-html-1.0.3.tgz",
       "integrity": "sha512-NiSupZ4OeuGwr68lGIeym/ksIZMJodUGOSCZ/FSnTxcrekbvqrgdUxlJOMpijaKZVjAJrWrGs/6Jy8OMuyj9ow=="
     },
+    "node_modules/escape-string-regexp": {
```
Comment (sotoJ24) [inline package-lock.json:329]: Remove -> ` 
 "node_modules/escape-string-regexp": {
      "version": "1.0.5",
      "resolved": "https://registry.npmjs.org/escape-string-regexp/-/escape-string-regexp-1.0.5.tgz",
      "integrity": "sha512-vbRorB5FUQWvla16U8R/qgaFIya2qGzwDrNmCZuYKrbdSUMG6I1ZCGQRefkRVhuOkIGVne7BQ35DSfo1qvJqFg==",
      "engines": {
        "node": ">=0.8.0"
      }
    },
`

File: package-lock.json [hunk 6]
```diff
@@ -370,6 +419,14 @@
         "node": ">= 0.8"
       }
     },
+    "node_modules/find": {
```
Comment (sotoJ24) [inline package-lock.json:422]: Remove -> 

`
 "node_modules/find": {
      "version": "0.3.0",
      "resolved": "https://registry.npmjs.org/find/-/find-0.3.0.tgz",
      "integrity": "sha512-iSd+O4OEYV/I36Zl8MdYJO0xD82wH528SaCieTVHhclgiYNe9y+yPKSwK+A7/WsmHL1EZ+pYUJBXWTL5qofksw==",
      "dependencies": {
        "traverse-chain": "~0.1.0"
      }
    },
`

File: package-lock.json [hunk 7]
```diff
@@ -391,6 +448,27 @@
         "node": ">= 0.6"
       }
     },
+    "node_modules/fs-extra": {
```
Comment (sotoJ24) [inline package-lock.json:451]: Remove -> 

`
"node_modules/fs-extra": {
      "version": "8.1.0",
      "resolved": "https://registry.npmjs.org/fs-extra/-/fs-extra-8.1.0.tgz",
      "integrity": "sha512-yhlQgA6mnOJUKOsRUFsgJdQCvkKhcz8tlZG5HBQfReYZy46OwLcY+Zia0mtdHsOo9y/hP+CxMN0TU9QxoOtG4g==",
      "dependencies": {
        "graceful-fs": "^4.2.0",
        "jsonfile": "^4.0.0",
        "universalify": "^0.1.0"
      },
      "engines": {
        "node": ">=6 <7 || >=8"
      }
    },
    "node_modules/fs-extra/node_modules/jsonfile": {
      "version": "4.0.0",
      "resolved": "https://registry.npmjs.org/jsonfile/-/jsonfile-4.0.0.tgz",
      "integrity": "sha512-m6F1R3z8jjlf2imQHS2Qez5sjKWQzbuuhuJ/FKYFRZvPE3PuHcSMVZzfsLhGVOkfd20obL5SWEBew5ShlquNxg==",
      "optionalDependencies": {
        "graceful-fs": "^4.1.6"
      }
    },
`

File: package-lock.json [hunk 8]
```diff
@@ -428,6 +506,11 @@
         "url": "https://github.com/sponsors/ljharb"
       }
     },
+    "node_modules/graceful-fs": {
```
Comment (sotoJ24) [inline package-lock.json:509]: Remove -> 

`    
"node_modules/graceful-fs": {
      "version": "4.2.11",
      "resolved": "https://registry.npmjs.org/graceful-fs/-/graceful-fs-4.2.11.tgz",
      "integrity": "sha512-RbJ5/jmFcNNCcDV5o9eTnBLJ/HszWV0P73bc+Ff4nS/rJj+YaS6IGyiOL0VoBYX+l1Wrl3k63h/KrH+nhJ0XvQ=="
    },
`

File: package-lock.json [hunk 9]
```diff
@@ -448,6 +531,14 @@
         "graphql": "14 - 16"
       }
     },
+    "node_modules/has-flag": {
```
Comment (sotoJ24) [inline package-lock.json:534]: Remove -> 
`   "node_modules/has-flag": {
      "version": "3.0.0",
      "resolved": "https://registry.npmjs.org/has-flag/-/has-flag-3.0.0.tgz",
      "integrity": "sha512-sKJf1+ceQBr4SMkvQnBDNDtf4TXpVhVGateu0t918bl30FnbE2m4vNLX+VWe/dpjlb+HugGYzW7uQXH98HPEYw==",
      "engines": {
        "node": ">=4"
      }
    },
`

File: package-lock.json [hunk 10]
```diff
@@ -547,11 +638,40 @@
         "url": "https://github.com/sponsors/sindresorhus"
       }
     },
+    "node_modules/is-wsl": {
```
Comment (sotoJ24) [inline package-lock.json:641]: Remove -> 

` "node_modules/is-wsl": {
      "version": "1.1.0",
      "resolved": "https://registry.npmjs.org/is-wsl/-/is-wsl-1.1.0.tgz",
      "integrity": "sha512-gfygJYZ2gLTDlmbWMI0CE2MwnFzSN/2SZfkMlItC4K/JBlsWVDB0bO6XhqcY13YXE7iMcAJnzTCJjPiTeJJ0Mw==",
      "engines": {
        "node": ">=4"
      }
    },
    "node_modules/js-base64": {
      "version": "2.6.4",
      "resolved": "https://registry.npmjs.org/js-base64/-/js-base64-2.6.4.tgz",
      "integrity": "sha512-pZe//GGmwJndub7ZghVHz7vjb2LgC1m8B07Au3eYqeqv9emhESByMXxaEgkUkEqJe87oBbSniGYoQNIBklc7IQ=="
    },
    "node_modules/jsonfile": {
      "version": "5.0.0",
      "resolved": "https://registry.npmjs.org/jsonfile/-/jsonfile-5.0.0.tgz",
      "integrity": "sha512-NQRZ5CRo74MhMMC3/3r5g2k4fjodJ/wh8MxjFbCViWKFjxrnudWSY5vomh+23ZaXzAS7J3fBZIR2dV6WbmfM0w==",
      "dependencies": {
        "universalify": "^0.1.2"
      },
      "optionalDependencies": {
        "graceful-fs": "^4.1.6"
      }
    },
`

File: package-lock.json [hunk 11]
```diff
@@ -547,11 +638,40 @@
...
+      "optionalDependencies": {
+        "graceful-fs": "^4.1.6"
+      }
+    },
     "node_modules/kuler": {
       "version": "2.0.0",
       "resolved": "https://registry.npmjs.org/kuler/-/kuler-2.0.0.tgz",
       "integrity": "sha512-Xq9nH7KlWZmXAtodXDDRE7vs6DU1gTU8zYDHDiWLSip45Egwq3plLHzPn27NgvzL2r1LMPC1vdqh98sQxtqj4A=="
     },
+    "node_modules/lodash": {
```
Comment (sotoJ24) [inline package-lock.json:670]: Remove ->

`    "node_modules/lodash": {
      "version": "4.17.21",
      "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.21.tgz",
      "integrity": "sha512-v2kDEe57lecTulaDIuNTPy3Ry4gLGJ6Z1O3vE1krgXZNrsQ+LFTGHVxVjcXPs17LhbZVGedAJv8XZ1tvj5FvSg=="
    },`

File: package-lock.json [hunk 12]
```diff
@@ -640,6 +760,14 @@
         "node": ">= 0.6"
       }
     },
+    "node_modules/node-emoji": {
```
Comment (sotoJ24) [inline package-lock.json:763]: Remove -> 
`    "node_modules/node-emoji": {
      "version": "1.11.0",
      "resolved": "https://registry.npmjs.org/node-emoji/-/node-emoji-1.11.0.tgz",
      "integrity": "sha512-wo2DpQkQp7Sjm2A0cq+sN7EHKO6Sl0ctXeBdFZrL9T9+UywORbufTcTZxom8YqpLQt/FqNMUkOpkZrJVYSKD3A==",
      "dependencies": {
        "lodash": "^4.17.21"
      }
    },`

File: package-lock.json [hunk 13]
```diff
@@ -678,6 +806,17 @@
         "fn.name": "1.x.x"
       }
     },
+    "node_modules/open": {
```
Comment (sotoJ24) [inline package-lock.json:809]: Remove ->

`    "node_modules/open": {
      "version": "6.4.0",
      "resolved": "https://registry.npmjs.org/open/-/open-6.4.0.tgz",
      "integrity": "sha512-IFenVPgF70fSm1keSd2iDBIDIBZkroLeuffXq+wKTzTJlBpesFWojV9lb8mzOfaAzM1sr7HQHuO0vtV0zYekGg==",
      "dependencies": {
        "is-wsl": "^1.1.0"
      },
      "engines": {
        "node": ">=8"
      }
    },`

File: package-lock.json [hunk 14]
```diff
@@ -904,6 +1043,17 @@
         "safe-buffer": "~5.2.0"
       }
     },
+    "node_modules/supports-color": {
```
Comment (sotoJ24) [inline package-lock.json:1046]: Remove ->

`    "node_modules/supports-color": {
      "version": "5.5.0",
      "resolved": "https://registry.npmjs.org/supports-color/-/supports-color-5.5.0.tgz",
      "integrity": "sha512-QjVjwdXIt408MIiAqCX4oUKsgU2EqAGzs2Ppkm4aQYbjm+ZEWEcW4SfFNTr4uMNZma0ey4f5lgLrkB0aX0QMow==",
      "dependencies": {
        "has-flag": "^3.0.0"
      },
      "engines": {
        "node": ">=4"
      }
    },`

File: package-lock.json [hunk 15]
```diff
@@ -917,6 +1067,11 @@
         "node": ">=0.6"
       }
     },
+    "node_modules/traverse-chain": {
```
Comment (sotoJ24) [inline package-lock.json:1070]: Remove ->

`    "node_modules/traverse-chain": {
      "version": "0.1.0",
      "resolved": "https://registry.npmjs.org/traverse-chain/-/traverse-chain-0.1.0.tgz",
      "integrity": "sha512-up6Yvai4PYKhpNp5PkYtx50m3KbwQrqDwbuZP/ItyL64YEWHAvH6Md83LFLV/GRSk/BoUVwwgUzX6SOQSbsfAg=="
    },`

File: package-lock.json [hunk 16]
```diff
@@ -937,6 +1092,14 @@
         "node": ">= 0.6"
       }
     },
+    "node_modules/universalify": {
```
Comment (sotoJ24) [inline package-lock.json:1095]: Remove ->
`   "node_modules/universalify": {
      "version": "0.1.2",
      "resolved": "https://registry.npmjs.org/universalify/-/universalify-0.1.2.tgz",
      "integrity": "sha512-rBJeI5CXAlmy1pV+617WB9J63U6XcazHHF2f2dbJix4XzpUF0RS3Zbj0FGIOCAva5P/d/GBOYaACQ1w+0azUkg==",
      "engines": {
        "node": ">= 4.0.0"
      }
    },
`

File: package-lock.json [hunk 17]
```diff
@@ -958,6 +1121,15 @@
         "node": ">= 0.4.0"
       }
     },
+    "node_modules/uuid": {
```
Comment (sotoJ24) [inline package-lock.json:1124]: Remove -> 

`    "node_modules/uuid": {
      "version": "3.4.0",
      "resolved": "https://registry.npmjs.org/uuid/-/uuid-3.4.0.tgz",
      "integrity": "sha512-HjSDRw6gZE5JMggctHBcjVak08+KEVhSIiDzFnT9S9aegmp85S/bReBVTb4QTFaRNptJ9kuYaNhnbNEOkbKb/A==",
      "deprecated": "Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.",
      "bin": {
        "uuid": "bin/uuid"
      }
    },`

**Review by sotoJ24 — CHANGES_REQUESTED** [review]
